src/processors/universal.ts

```diff
@@ -27,22 +28,34 @@ import { approveToken } from "./utils/approve";
 export type UniversalProcessorParams = {
   minProfitabilityBps: number;
   mempoolInterval: number;
+  batchUnlockSize: number;
 };
 
 class UniversalProcessor extends BaseOrderProcessor {
   private mempoolService: MempoolService;
   private priorityQueue = new Set<string>();
   private queue = new Set<string>();
-  private ordersMap = new Map<string, IncomingOrderContext>();
-
+  private incomingOrdersMap = new Map<string, IncomingOrderContext>(); // key orderid
```

Comment:
Please give meaningful comments of what is stored here and where are references to these objects are defined. It is not clear that `priorityQueue` and `queue` store references to `incomingOrdersMap`, while `unlockBatchesOrderIdMap` store references to `ordersDataMap`.